llava/README.md

```diff
@@ -0,0 +1,41 @@
+# LLaVA
+
+An example of LLaVA: Large Language and Vission Assistant in MLX. LLlava is a multi-modal model that can generate text from images and text prompts. [^1]
+
+## Setup:
+
+Install the dependencies:
+
+```bash
+pip install -r requirements.txt
+```
+
+## Run
```

Comment:
Would we mention that the can use the generate.py script as the cli as well? That may make it easier for them to test things out without having to code the script.